examples/demo-rollup/tests/evm/uniswap/hardhat.config.js

```diff
@@ -12,4 +12,9 @@ module.exports = {
       },
     }
   },
+  networks: {
+    sov: {
+      url: "http://127.0.0.1:12345"
```

Comment:
info: Default port for ethereum rpc is 8545

Reply:
Yes that's why we defined a separate sov network.